src/dstack/_internal/server/services/projects.py

```diff
@@ -497,8 +596,102 @@ def _is_project_admin(
     user: UserModel,
     project: ProjectModel,
 ) -> bool:
+    # Check if user is the project owner
```

Comment:
@haydnli-shopify, once again, please remove all redundant comments that duplicate the code. We won't merge PRs with comments clutter.

If a function becomes hard to follow, better refactor the code into several appropriately named functions.

https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/#h2-b7593c698a940